pkgs/by-name/ci/cinny-desktop/package.nix

```diff
@@ -14,26 +13,30 @@
   dbus,
   glib,
   glib-networking,
-  libayatana-appindicator,
   webkitgtk_4_0,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "cinny-desktop";
   # We have to be using the same version as cinny-web or this isn't going to work.
-  version = "4.2.3";
+  version = "4.3.0";
 
   src = fetchFromGitHub {
     owner = "cinnyapp";
     repo = "cinny-desktop";
     tag = "v${version}";
-    hash = "sha256-yNGzgkZXz/VroGGnZFqo5n2v3cE6/tvpQv5U4p27row=";
+    hash = "sha256-qmgQV5WRb1rFFPIWiI4tLC+CB264y4/qQS5kDwQOaCk=";
   };
 
   sourceRoot = "${src.name}/src-tauri";
 
+  # Temporary hack: https://github.com/cinnyapp/cinny-desktop/issues/415
```

Comment:
Should be unnecessary in v4.3.1 if https://github.com/cinnyapp/cinny-desktop/pull/417 works, should test if patch file is unnecessary